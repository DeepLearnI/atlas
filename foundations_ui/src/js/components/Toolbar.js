import React, { Component } from 'react';

class Toolbar extends Component {
  render() {
    return (
      <div className="toolbar-container">
        <div className="toolbar-home">
          <p className="toolbar-item toolbar-home-text">Home</p>
        </div>
        <div className="toolbar-support">
          <p className="toolbar-item toolbar-support-text">Support and Contact</p>
        </div>
      </div>
    );
  }
}

export default Toolbar;
