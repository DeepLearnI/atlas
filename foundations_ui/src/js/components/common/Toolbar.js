import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class Toolbar extends Component {
  render() {
    return (
      <div className="toolbar-container">
        <div className="job-header-logo-container">
          <Link to="/projects">
            <div className="i--icon-logo" />
          </Link>
          <h2 className="font-bold">Foundations</h2>
        </div>
        <ul className="header-links">
          <li>
            <Link to="/projects">Projects</Link>
          </li>
          <li>
            <a target="_blank" rel="noopener noreferrer" href="https://www.atlas.dessa.com/docs">Documentation</a>
          </li>
          <li>
            <Link to="/contact">Contact Support</Link>
          </li>
        </ul>
      </div>
    );
  }
}

export default Toolbar;
