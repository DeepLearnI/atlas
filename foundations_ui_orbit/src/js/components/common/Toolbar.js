import React, { Component } from "react";

const Toolbar = props => {
  return (
    <div className="toolbar-container">
      <p>Home</p>
      <p>Support and Contact</p>
    </div>
  );
};

export default Toolbar;
