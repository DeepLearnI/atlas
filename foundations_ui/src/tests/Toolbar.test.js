import React from 'react';
import ReactDOM from 'react-dom';
import Toolbar from '../js/components/common/Toolbar';
import { shallow } from 'enzyme';
import configureTests from './setupTests';

configureTests();

it('Shallow Renders Toolbar Home', () => {
  const wrapper = shallow(<Toolbar/>);
  expect(wrapper).toMatchSnapshot();
});