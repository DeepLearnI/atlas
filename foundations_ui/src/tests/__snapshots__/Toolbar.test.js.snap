// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Renders Toolbar Home 1`] = `
<div
  className="toolbar-container"
>
  <p
    className="font-regular"
  >
    Home
  </p>
  <p
    className="font-regular"
  >
    Support and Contact
  </p>
</div>
`;
